src/main/java/com/emc/mongoose/metrics/CustomMeter.java

```diff
@@ -1,19 +1,15 @@
 package com.emc.mongoose.metrics;
 
-import com.codahale.metrics.Clock;
-import com.codahale.metrics.EWMA;
-import com.codahale.metrics.Metric;
-
+import java.time.Clock;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.concurrent.atomic.LongAdder;
 
 import static java.lang.Math.exp;
 
-public class CustomMeter
-implements Metric {
+public class CustomMeter {
```

Comment:
please add the Meter interface for this

Reply:
but Meter is final and have own specific Constructor

Reply:
yep, Meter already exists, but I think it should be removed as far as it's needed only for exposing via JMX

Reply:
MeterMXBean may be removed too

Reply:
and what to do with JMX?

Reply:
yep